force-app/main/freelanceCommunity/lwc/hot_myServiceAppointments/hot_myServiceAppointments.js

```diff
@@ -7,6 +7,7 @@ import { formatRecord } from 'c/datetimeFormatter';
 
 export default class Hot_myServiceAppointments extends LightningElement {
     @track columns = [];
+    @track isNotEditable = true;
```

Comment:
Litt mye med 3 booleans for å utføre dette?

Booleans burde prøve å unngå NOT-ting.. isEditable er bedre enn isNotEditable